ckanext/canada/tables/choices/boolean.yaml

```diff
@@ -0,0 +1,6 @@
+'N':
```

Comment:
I might name this `yes_no.yaml` because to me (and to postgres) boolean means true/false

Reply:
Yeah makes sense, will update the "online_service_boolean.yaml" to "online_service_yes_no.yaml" as well.